### ISSUE TITLE
Using LR=1 causes immediate divergence (“snowflake” outputs) when training Flux-dev DiT with extra QKV branch

### ISSUE THREAD

**Issue by miaojiuchen** [konstmish/prodigy#39]
Hi, I am implementing a Flux-dev based image model and encountered an issue when using Prodigy optimizer.

**🔍 What I am trying to do**
I added an extra QKV branch for injecting additional conditions (following a paper ). [paper link](https://github.com/damo-cv/JCo-MVTON)
This extra branch adds roughly 1.61B parameters (~11% of total) to the DiT model.

**The whole original network was frozen and I only train these `extra_to_q` `extra_to_k` `extra_to_v` linears**

**⚠️ Problem**

When training with Prodigy using:
```
lr = 1.0
d0 = 2e-5
prodigy_use_bias_correction=True
prodigy_safeguard_warmup=True
```
the model immediately collapses, and generated images become pure noise (“snowflake images”) from the very beginning.

<img width="615" height="614" alt="Image" src="https://github.com/user-attachments/assets/48faa975-141e-4dc5-bde6-952f57a755c4" />

However, when I change only the learning rate:
```
lr = 2e-5
d0 = 2e-5
```
the model starts to learn normally and images gradually align with the training distribution.

<img width="3230" height="867" alt="Image" src="https://github.com/user-attachments/assets/3ffc286d-e9ec-4976-9d4d-07df30aae5a9" />

The training strategy was introduced in paper Section 5
https://arxiv.org/pdf/2508.17614

<img width="1082" height="207" alt="Image" src="https://github.com/user-attachments/assets/dc3f81d4-5ac6-4d8b-914e-bdb0af83b18c" />

---

❓ My questions

Is lr=1 reasonable for DiT / Transformer architectures when using Prodigy?
Many examples show lr=1, but those are often for Stable diffusion XL or smaller vision models.

I observed the curve of `d`, and it remained constant through all the training process.

`lr = self.optimizer.param_groups[0]['d']`

<img width="1086" height="586" alt="Image" src="https://github.com/user-attachments/assets/929266b2-8d44-4a12-a771-2c12f42f1f21" />

If needed, I can provide my training script

**Comment by miaojiuchen** [comment]
I feel this is a basic question, but it has really been bothering me for a long time. Really appreciate if anyone can give me some little hints ~
